_automating-configurations/default-use-cases.md

```diff
@@ -0,0 +1,200 @@
+| multi_modal_search                                   | Sets up an ingest pipeline with a text_image_embedding processor and a matching KNN index for multimodal search. User should provide a model ID                                                                                                                                   | model_id        |[pre-set-defaults-file](https://github.com/opensearch-project/flow-framework/blob/2.13/src/main/resources/defaults/bedrock-titan-embedding-defaults.json)                  |
+| multi_modal_search_with_bedrock_titan_multi_modal    | Deploys an Amazon Bedrock multimodal model and sets up an ingest pipeline with a text_image_embedding processor and a matching KNN index for multimodal search. User should provide there AWS Credentials                                                                         | AWS Credentials |[pre-set-defaults-file](https://github.com/opensearch-project/flow-framework/blob/2.13/src/main/resources/defaults/bedrock-titan-embedding-defaults.json)                  |
+| hybrid_search                                        | Sets up hybrid search by creating an ingest pipeline, a KNN index and a search pipeline with a normalization processor for appropriate usage. User should provide a model_id                                                                                                      | model_id        |[pre-set-defaults-file](https://github.com/opensearch-project/flow-framework/blob/2.13/src/main/resources/defaults/bedrock-titan-embedding-defaults.json)                  |
+| conversational_search_with_llm_deploy                | Deploys an LLM model (defaulted to cohere) and sets up a search pipeline with a retrieval_augmented_generation processor for use.                                                                                                                                                 | API Key         |[pre-set-defaults-file](https://github.com/opensearch-project/flow-framework/blob/2.13/src/main/resources/defaults/bedrock-titan-embedding-defaults.json)                  |
+
+
+### Addtional Examples
+
+Neural Sparse Use Case Tutorial Example:
+
+* One template we offer is neural sparse search with local pre-trained model 
+
+```
+POST /_plugins/_flow_framework/workflow?use_case=local_neural_sparse_search_bi_encoder
+
+```
+
+Response: 
+```
+{
+"workflow_id" : "8xL8bowB8y25Tqfenm50"
+}
+```
+
+
+#### The workflow created with the predefined defaults includes:
+
+1. Deploying one of our pretrained sprase encoding models (amazon/neural-sparse/opensearch-neural-sparse-encoding-v1 is default one here)
+2. Creates an ingest pipeline with a sparse encoding processor with the model deployed above
```

Comment:
Seems weird to have this be a header and a bulleted list. I think you can remove the header and have it be standard text.